3.3/.sti/bin/assemble

```diff
@@ -5,12 +5,42 @@ source .bashrc
 
 set -e
 
-echo "---> Installing application source"
+echo "---> Copying application source ..."
 cp -Rf /tmp/src/* ./
 
-echo "---> Building your Python application from source"
-if [ -f setup.py ]; then
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    pip install --user -r requirements.txt
+# If no requirements given, assume project root is installable
+# i.e., contains a valid `setup.py`.
```

Comment:
That's a false assumption, I'd rather leave the previous part which did the actual check for `setup.py`.